### PR TITLE
Website Update

### PR DESCRIPTION
**Pull request by shishkevichd**
### Changes
- Improved Russian translation
- Edited `i18n.js`
- Deleted `generic.html`
- Updated JQuery to 3.6.1

### CODE REVIEW

File: assets/js/i18n.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* 
     Internationalization for https://litebds.com
 
-    Maded by https://github.com/aye20054925
```
Comment (Jasonzyt) [inline assets/js/i18n.js:4]: It's not recommended to delete this.
You can add your homepage below.

File: assets/js/i18n.js [hunk 2]
```diff
@@ -95,6 +99,8 @@ class WebsiteInternationalization {
             "cpfllDocumentation": "文档",
             "pageNotSupported": "本页面与MOJANG STUDIOS、MICROSOFT与MINECRAFT论坛无关",
             "servers": "服务器统计",
+            "server_count": "{srv} 服务器",
+            "player_count": "{pl} 球员",
```
Comment (Jasonzyt) [inline assets/js/i18n.js:103]:
```suggestion
            "player_count": "{pl} 玩家",
```

File: assets/js/i18n.js [hunk 3]
```diff
@@ -122,6 +128,8 @@ class WebsiteInternationalization {
             "cpfllDocumentation": "文檔",
             "pageNotSupported": "本頁面與MOJANG STUDIOS、MICROSOFT與MINECRAFT論壇無關",
             "servers": "伺服器統計",
+            "server_count": "{srv} 服務器",
+            "player_count": "{pl} 球員",
```
Comment (Jasonzyt) [inline assets/js/i18n.js:132]:
```suggestion
            "player_count": "{pl} 玩家",
```

**Review by Jasonzyt — CHANGES_REQUESTED** [review]

File: assets/js/i18n.js (hunk 1 above)
Comment (shishkevichd) [reply assets/js/i18n.js:4]: > It's not recommended to delete this. You can add your homepage below.

I just change link (old link not working)

**Review by Jasonzyt — APPROVED** [review]
Thank you for your time and this contribution! 👍 😄

**Pull request merged**